media/json-jackson/src/main/java/org/glassfish/jersey/jackson/internal/jackson/jaxrs/cfg/EndpointConfigBase.java

```diff
@@ -53,7 +53,7 @@
     protected EndpointConfigBase(MapperConfig<?> config) {
         _config = config;
     }
-    
+
```

Comment:
I think formatting is not a purpose of the PR, so I would recommend not doing unnecessary formatting in files where there are no other changes.

Reply:
Yes,as I have already say, it was incorrect understanding of checkstyle rules.